packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.css

```diff
@@ -1,7 +1,15 @@
+:host {
+  --slds-c-toast-sizing-min-width: 100%;
+}
+
 .notification {
   background-color: var(---lwc-colorBackgroundAlt, white);
   border: 1px solid var(--lwc-colorBackgroundToast, rgb(116, 116, 116));
   margin-right: 0;
   margin-left: 0;
   width: 100%;
+}
+
+.notification-container {
+  z-index: 1;
```

Comment:
I personally would put this at like 10?
So at least there can be content in between 0 and the notifications?

Reply:
modal is 40 so we could do steps of 20 and set it at 20, but I didnt think there would be content between 0 and 1